test/load/README.md

```diff
@@ -1,2 +1,32 @@
 ## Terraform Consul Load Testing
```

Comment:
The header sizes are reversed in the article: this should start with `#` and the sections after should start with `##`.